src/vignette/api/legacy/routes.clj

```diff
@@ -49,6 +50,20 @@
                   :image-type #"images"
                   :original original-regex}))
 
+(def interactive-maps-route
+  (route-compile "/:wikia:path-prefix/:original"
+                 {:wikia interactive-maps-regex
+                  :path-prefix #"/\d/\d|"
+                  :original original-regex}))
+
+(def interactive-maps-thumbnail-route
+  (route-compile "/:wikia/thumb/:original/:dimension:offset:thumbname"
```

Comment:
Not that I'm aware of. @rogatty do you know if there can be directories between `thumb` and `20150205173220%21phpZDfa00.jpg` in this image:

`/intmap_tile_set_4823/thumb/20150205173220%21phpZDfa00.jpg/1110x300x5-20150205173220%21phpZDfa00.jpg`

If so then we'll need to add the `:path-prefix`.